tron/tron-normalization/src/data.ts

```diff
@@ -0,0 +1,92 @@
+export interface BlockHeader {
+    height: number
+    hash: string
+    parentHash: string
+    txTrieRoot: string
+    version?: number
+    timestamp: number
+    witnessAddress: string
+    witnessSignature?: string
+}
+
+
+export interface Log {
+    transactionIndex: number
+    logIndex: number
+    address: string
+    data?: string
+    topics: string[]
+}
+
+
+export interface TransactionResult {
+    contractRet?: string
+}
+
+
+export interface Transaction {
+    transactionIndex: number
+    hash: string
+    ret?: TransactionResult[]
+    signature?: string[]
+    type: string
+    parameter: any
+    permissionId?: number
+    refBlockBytes?: string
+    refBlockHash?: string
+    feeLimit?: bigint
+    expiration?: number
+    timestamp?: number
+    rawDataHex: string
+    fee?: bigint
+    contractResult?: string
+    contractAddress?: string
+    resMessage?: string
+    withdrawAmount?: bigint
+    unfreezeAmount?: bigint
+    withdrawExpireAmount?: bigint
+    cancelUnfreezeV2Amount?: Record<string, bigint>
+    result?: string
+    energyFee?: bigint
+    energyUsage?: bigint
+    energyUsageTotal?: bigint
+    netUsage?: bigint
+    netFee?: bigint
+    originEnergyUsage?: bigint
+    energyPenaltyTotal?: bigint
+    _transferContractOwner?: string
```

Comment:
I don't think, that introduction of this fields is justified. Their derivation is quite strait forward a can be done in-place.